tests/test_models/test_simple_models.py

```diff
@@ -839,3 +839,19 @@ def test_deadline_ma_predict_components_sum_up_to_target(
     components = forecast.get_target_components()
 
     np.testing.assert_allclose(target.values, components.sum(axis=1, level="segment").values)
+
+
+@pytest.mark.parametrize(
+    "method_name, expected_values",
+    (("forecast", [[15, 2], [16, 4], [17, 6]]), ("predict", [[15, 2], [16, 4], [17, 6]])),
```

Comment:
Try to come up with the case where the expected values are different for forecast and predict -- i.e forecast should use predicted value as lag and predict uses real value